### ISSUE TITLE
A/B test IOU invite notifications 

### ISSUE THREAD

**Issue by danielrvidal** [Expensify/App#47108]
Problem: Users often get notifications from Expensify and never log in. We know this happens, but we don’t know how often it happens because we don’t track it. Thus, we don’t know how big of a problem this actually is, which is a problem in itself.

Solution: Start measuring and a/b testing CRITICAL emails that drive conversion of new users. We had a [long thread](https://expensify.slack.com/archives/C036QM0SLJK/p1719877033214699?thread_ts=1714767525.501349&cid=C036QM0SLJK) where we ideated on a new email format for invites (and in general) that look much more like the full chat UI than our current emails. The bottom-up flow for IOUs is our most important invite email because it brings in the decision maker who can bring on more users. Thus, let’s create a baseline of how often people are opening and clicking the CTA in the current email (image 1), and then let’s test the updated version (image 2) to validate the proposed email format converts better. If it does, we’d look to update all of our emails with this formatting.

![image](https://github.com/user-attachments/assets/f9df28e2-a670-444f-9a33-02b4a081061a)

![image](https://github.com/user-attachments/assets/0483e018-8424-48eb-932e-3acde1216053)

**Comment by justinpersaud** [comment]
Sure, I am prioritizing my time with fighting spam in #fireroom-2024-05-23-spam right now but I should be able to help with the A/B stuff on the side. Anu was already asking some questions about it.

**Comment by shawnborton** [comment]
Just let me know how I can be helpful with mocks!

**Comment by danielrvidal** [comment]
Sounds good, I'll work with @shawnborton early next week!

**Comment by danielrvidal** [comment]
I was talking with @jamesdeanexpensify and @trjExpensify about how our proposal that landed in that thread is actually slightly different from the text currently live in the product. 

The text that is in the proposed mock is from this doc: https://docs.google.com/document/d/19TNWIrcPh4X-sr5ycWYYb-LfSPAb8fVx005asviM2II/edit?pli=1#heading=h.sv15vxk1s1ee 

So I think @jamesdeanexpensify is going to create a GH to go forward with those updates. So we can assume the UI from the DM is going to use the text that is in the proposed mock included. So I don't know if we need new mocks, we can just go with the one @shawnborton actually already created.

**Comment by danielrvidal** [comment]
@shawnborton @jamesdeanexpensify @anmurali I'm thinking we should pause this one until @jamesdeanexpensify gets the copy updated for the DM chat. Otherwise we'd be testing something that is going to change. 

@jamesdeanexpensify is working on that here: https://github.com/Expensify/App/issues/47427

I'll ask @jamesdeanexpensify what the next steps are on that issue though just so we can align on timing/expectations.

**Comment by shawnborton** [comment]
That works for me.

**Comment by danielrvidal** [comment]
Waiting on the f1 room to open then I'm going to drop an update in there.

**Comment by danielrvidal** [comment]
I created an updated here. I'll be creating the IOU here shortly:
https://expensify.slack.com/archives/C07HPDRELLD/p1724245865161379

**Comment by danielrvidal** [comment]
We're getting the text updated so we'll start working on the updates this week.

**Comment by jamesdeanexpensify** [comment]
@danielrvidal we might need to push a bit harder on getting [this issue](https://github.com/Expensify/App/issues/47427) assigned (either in #whatsnext, the F1 room, or both). And maybe we should make it a `Daily`?

**Comment by danielrvidal** [comment]
Yea, we need to figure out who is curating the room so this could be called out as unassigned in the weekly update. Then we could also tag hot picks to make the engineers more aware.

**Comment by danielrvidal** [comment]
Moving this to weekly as we need to get the beginningOfChatHistory updated first.

**Comment by danielrvidal** [comment]
We're still working on the copy for this. I'm bringing a p/s for the a/b testing tomorrow.

**Comment by anmurali** [comment]
Discussing [here](https://expensify.slack.com/archives/C07HPDRELLD/p1725893690431599)

**Comment by danielrvidal** [comment]
We're still making progress, we should have the HL DD out on Mon/Tuesday of next week.

**Comment by danielrvidal** [comment]
Design doc is going out today I think.

**Comment by danielrvidal** [comment]
This is in review. Moving to weekly.

**Comment by danielrvidal** [comment]
I sent out the doc two days ago,

**Comment by danielrvidal** [comment]
Detailed is going out shortly for the trackingKey's so we'll be conducting this soon.

**Comment by danielrvidal** [comment]
We're still working on this. Subject lines detailed is being done.

**Comment by danielrvidal** [comment]
Subject lines test is getting deployed on Monday.

**Comment by anmurali** [comment]
Latest is [here](https://expensify.slack.com/archives/C07HPDRELLD/p1730161250824289)
@danielrvidal is testing it.

**Comment by danielrvidal** [comment]
This is still being discussed here: https://expensify.slack.com/archives/C07HPDRELLD/p1733262477107509?thread_ts=1730161250.824289&cid=C07HPDRELLD

**Comment by danielrvidal** [comment]
Our latest update here: https://expensify.slack.com/archives/C07HPDRELLD/p1733862564966199?thread_ts=1730161250.824289&cid=C07HPDRELLD

@anmurali is creating another issue to store tags as NVPs.

**Comment by danielrvidal** [comment]
We're waiting for data

**Comment by danielrvidal** [comment]
I think we still need the data to start populating. We had a huge dropoff over break so I'm waiting for the data to populate since coming back.

**Comment by danielrvidal** [comment]
Closing this as we're not tracking it.
